pkgs/data/fonts/u001/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib, stdenvNoCC, fetchzip }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "u001";
+  version = "unstable-2016-08-01"; # date in the zip file, actual creation date unknown
+
+  src = fetchzip {
+    url = "https://fontlibrary.org/assets/downloads/u001/3ea00b3c0c8fa6ce4373e5766fafd651/u001.zip";
+    sha256 = "sha256-7H32pfr0g68XP5B48VUY99e6fbd7rhH6fEnCKNXWEkU=";
+    stripRoot = false;
+  };
+
+  dontBuild = true;
+
+  # We’ll adjust the nonstandard naming convention here
+  installPhase = ''
+    runHook preInstall
+
+    TTF_DIR=$out/share/fonts/truetype
+    DOC_DIR=$out/share/doc
+
+    mkdir -p $TTF_DIR $DOC_DIR
+
+    cp u001-reg.ttf $TTF_DIR/U001-Regular.ttf
+    cp u001-ita.ttf $TTF_DIR/U001-Italic.ttf
+    cp u001-bol.ttf $TTF_DIR/U001-Bold.ttf
+    cp u001-bolita.ttf $TTF_DIR/U001-BoldItalic.ttf
+    cp u001con-reg.ttf $TTF_DIR/U001Condensed-Regular.ttf
+    cp u001con-ita.ttf $TTF_DIR/U001Condensed-Italic.ttf
+    cp u001con-bol.ttf $TTF_DIR/U001Condensed-Bold.ttf
+    cp u001con-bolita.ttf $TTF_DIR/U001Condensed-BoldItalic.ttf
+
+    cp Copying.AFPL.txt $DOC_DIR
```

Comment:
There is no necessity of including license files.
The `meta.license` fulfills this role.

Reply:
IMHO the only justifiable exceptions should be

1. the build script shipped with the source code installs the license file; and/or
2. the license file is used by the program in runtime (think on an ABOUT button)